dotnetv3/cross-service/AuroraItemTracker/Readme.md

```diff
@@ -0,0 +1,180 @@
+
+## Build the code
+
+This application has two parts: a user interface that uses React.js and a
+RESTful API created with C# and .NET 6. The React user interface is a single-page
+application (SPA) that interacts with the C# RESTful API by making `GET`, `PUT`, and
+`POST` requests.
+
+### RESTful API
+
+The API uses the [AmazonRDSDataServiceClient](https://docs.aws.amazon.com/sdkfornet/v3/apidocs/items/RDSDataService/TRDSDataServiceClient.html)
+object to perform CRUD operations on an Aurora Serverless database. The API
+returns JSON data in an HTTP response, as shown in the following illustration.
+
+![AWS Tracker JSON response](images/aurora_item_tracker_response.png)
+
+#### Configure the application
+Before you run the .NET application, set the configuration values for your Aurora serverless
+database, and your verified email address in the `appsettings.json` file. Alternatively, add an `appsettings.development.json` file
+with your local settings.
+
+#### Run the application
+After the example compiles, you can run it from the command line. To do so,
+navigate to the folder that contains the .csproj file and run the following
+command:
+
+```
+dotnet run
+```
+
+Alternatively, you can run the example from within your IDE.
+
+#### Tests
+
+⚠️ Running the tests might result in charges to your AWS account.
+
+The solution includes a test project. To run the tests, navigate to the folder that contains the test project and then issue the following command:
+
+```
+dotnet test
+```
+
+Alternatively, you can open the example solution and use the Visual Studio Test Runner to run the tests.
+
+### Aurora Item Tracker user interface
+To start the React web application, you can download files from the following GitHub repository. Included in this repository are instructions on how to set up the project. Click the following link to access the GitHub location [Item Tracker web client](https://github.com/awsdocs/aws-doc-sdk-examples/tree/main/resources/clients/react/item-tracker/README.md).  
+
+When the web application is running, you will see something like the following.
+
+![Aurora Item Tracker UI](images/elapp1.png)
+
+#### Use the React web application
+
+A user can perform these tasks using the web application:
+
+1. View all items
+1. View active items only
+1. View archived items only
+1. Add a new item
+1. Change an active item into an archived item.
+1. Send a report as an email attachment
+
+The web application displays *active*, *archived*, or all items. For example, the following illustration shows the React application displaying active data.
+
+![Viewing active items](images/elapp2.png)
+
+Here is the React application displaying archived data.
+
+![Viewing archived items](images/elapp3.png)
+
+The user can insert a new item into the **items** table from the user interface in the React front end application. In the following image, you can see the new item form. 
```

Comment:
**items** -> **work_items**
(assuming you want to use the name of the table, otherwise I think it's fine to use plain English, like "work item table".)